pygount/analysis.py

```diff
@@ -27,7 +27,11 @@
 from pygount.common import mapped_repr
 from pygount.git_storage import GitStorage, git_remote_url_and_revision_if_any
 
-GIT_REPO_REGEX = re.compile(r"^(https?://|git@)")
+HAS_URL_PREFIX = re.compile(r"^(https?://)")
```

Comment:
Naming: It's not a boolean rather a regex, so: `URL_REGEX`.

We might also want to be a bit more precise because a log of things can be an URL, so possibly even better: `HTTP_URL_REGEX`